modules/compatibility/src/test/java/org/apache/ignite/compatibility/persistence/SnapshotCompatibilityTest.java

```diff
@@ -142,19 +135,35 @@ public void setUp() {
     /** */
     @Test
     public void testSnapshotRestore() throws Exception {
+        assumeFalse("User must not restore both snapshots and cache dumps simultaneously", incSnp && cacheDump);
+
+        // See <a href="https://issues.apache.org/jira/browse/IGNITE-23222">...</a>
+        assumeFalse("Restoring incremental snapshots if consistent ID is null is fixed in 2.17.0", incSnp && !customConsId);
```

Comment:
"Restoring incremental snapshots if consistent ID is null is fixed in 2.17.0" -> "https://issues.apache.org/jira/browse/IGNITE-23222"